x-pack/plugins/security_solution/public/detection_engine/rule_gaps/components/rule_backfills_info/index.tsx

```diff
@@ -148,23 +184,39 @@ export const RuleBackfillsInfo = React.memo<{ ruleId: string }>(({ ruleId }) =>
     }
   };
 
+  const handleRefresh = () => {
```

Comment:
Should we use `useCallback` for this and other methods (`getBackfillRowsFromResponse`, `getBackfillsTableColumns`, `handleRefreshChange` and `handleTableChange`)?

Also, should we wrap `pagination` in `useMemo`?

Reply:
Hey, not sure we should do this, as we pass those props directly to EUI components, and if I understand correctly those components not wrapped to memo, meaning they still rerender if parent component rerender